content/docs/connect/connection-errors.md

```diff
@@ -186,4 +187,12 @@ Error querying the database: db error: ERROR: prepared statement
 
 Prisma Migrate requires a direct connection to the database. It does not support a pooled connection with PgBouncer, which is the connection pooler used by Neon. Attempting to run Prisma Migrate commands, such as `prisma migrate dev`, with a pooled connection causes this error. To resolve this issue, please refer to our [Prisma Migrate with PgBouncer](/docs/guides/prisma-migrate#prisma-migrate-with-pgbouncer) instructions.
 
+## Terminating connection due to administrator command
+
+The `terminating connection due to administrator command` error is typically encountered when running a query from a connection that has sat idle long enough for the compute endpoint to suspend due to inactivity. Neon automatically suspends a compute endpoint after 5 minutes of inactivity, by default. You can reproduce this error by connecting to your database from an application or client such as `psql`, letting the connection remain idle until the compute suspends, and then running a query from the same connection.
+
+This error may also occur when a connection is established less than 500ms before a compute endpoint suspends due to inactivity. Neon monitors for activity every 500ms. In this case, Neon is not aware of the new activity and proceeds with the compute suspension operation, resulting in the connection being terminated.
```

Comment:
I've just learned that there are more cases to that, e.g. any connection shorter then 500ms will not extend endpoint lifetime. That explains why we see this error often (serverless driver does short connections). Not sure if we should document that. I'll try to get that fixed soon.

Reply:
OK. Thanks. I will leave this one out. We can add it back if needed.